Typescript with moduleResolution node16/nodenext is broken

**Description**
Describe the problem and how to reproduce it. If appropriate, include code samples, screenshots, error messages, etc. 

Building a Typescript project using packages of `"type": "module"` and `"moduleResolution": "nodenext"` is broken because the types in the `exports` section is not specified. 

The issue is discussed here: https://github.com/microsoft/TypeScript/issues/50466 

**Environment:**
Specify the environment where you are witnessing the problem:

Version 3.0.22, ESM, Node 16, Typescript, MacOS

**Details**
Add any other information, context or details that could help track down the problem.

Since Typescript does not find types with the `webmidi` project, it reverts to the `@types/webmidi`, imported by the `jzz` package (a dependency of webmidi). These are probably old. 

```
frontend/audio-ui/src/events/midi-event.ts:1:39 - error TS2306: File '/Users/torgeir/dev/tunasong/node_modules/@types/webmidi/index.d.ts' is not a module.

1 import type { NoteMessageEvent } from 'webmidi'
                                        ~~~~~~~~~
```

To reproduce: 

1. git clone https://github.com/thovden/webmidi-client-import-issue.git
2. npm i
3. tsc

The fix is easy and I will submit a PR for the fix shortly.

--- a/package.json
+++ b/package.json
@@ -5,12 +5,15 @@
   "main": "./dist/cjs/webmidi.cjs.min.js",
   "types": "./dist/cjs/webmidi.cjs.d.ts",
   "exports": {
-    "require": {
-      "types": "./dist/cjs/webmidi.cjs.d.ts",
-      "default": "./dist/cjs/webmidi.cjs.min.js",
-    "import": {
-      "types": "./dist/esm/webmidi.esm.d.ts",
-      "default": "./dist/esm/webmidi.esm.js"
+    ".": {
+      "require": {
+        "types": "./dist/cjs/webmidi.cjs.d.ts",
+        "default": "./dist/cjs/webmidi.cjs.min.js",
+      },
+      "import": {
+        "types": "./dist/esm/webmidi.esm.d.ts",
+        "default": "./dist/esm/webmidi.esm.js"
+      }
     }
   },
   "author": {

--- a/package.json
+++ b/package.json
@@ -5,8 +5,13 @@
   "main": "./dist/cjs/webmidi.cjs.min.js",
   "types": "./dist/cjs/webmidi.cjs.d.ts",
   "exports": {
-    "require": "./dist/cjs/webmidi.cjs.min.js",
-    "import": "./dist/esm/webmidi.esm.min.js"
+    "require": {
+      "types": "./dist/cjs/webmidi.cjs.d.ts",
+      "default": "./dist/cjs/webmidi.cjs.min.js",
+    "import": {
+      "types": "./dist/esm/webmidi.esm.d.ts",
+      "default": "./dist/esm/webmidi.esm.js"
+    }
   },
   "author": {
     "name": "Jean-Philippe Côté",

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     ".": {
       "require": {
         "types": "./dist/cjs/webmidi.cjs.d.ts",
-        "default": "./dist/cjs/webmidi.cjs.min.js",
+        "default": "./dist/cjs/webmidi.cjs.min.js"
       },
       "import": {
         "types": "./dist/esm/webmidi.esm.d.ts",

